test/E2ETest/WebStack.QA.Test.OData/ODataOrderByTest/OrderByDataModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace WebStack.QA.Test.OData.ODataOrderByTest
+{
+    public class Item
+    {
+        [Key]
+        [Column(Order = 2)]
```

Comment:
What is the behavior if [Key] but not [Column]? I suspect that key gets ordered first, which may not be the correct behavior?

Reply:
@robward-ms  Files Updated. I was took logic from the bug. At this moment default "order by", by property name.
If need another logic please describe.

Reply:
@robward-ms Local build successful.